subprojects/core/src/integTest/groovy/org/gradle/configuration/project/LifecycleProjectEvaluatorIntegrationTest.groovy

```diff
@@ -151,6 +156,11 @@ class LifecycleProjectEvaluatorIntegrationTest extends AbstractIntegrationSpec {
             children*.displayName == ["Apply script after.gradle to project ':foo'"]
             parentId == configOp.id
         }
+        with(operations.only(NotifyTaskGraphWhenReadyBuildOperationType, { it.details.buildPath == ':' })) {
```

Comment:
Would be worth some composite build coverage, and buildSrc.

Reply:
added. Interestingly (to keep in mind) the taskgraph when ready evaluation of the included build is optional, depending on if any task of the included build is triggered or not